charts/alertmanager/templates/alertmanager-sidecar-configmap.yaml

```diff
@@ -0,0 +1,41 @@
+################################
+## Alertmanager sidecar ConfigMap
```

Comment:
```suggestion
## Alertmanager auth sidecar ConfigMap
```

And filename `alertmanager-auth-sidecar-configmap.yaml`.

Reply:
fixed via e670326f1432f6ed151c598e8e6c663dc648a322

Reply:
@andriisoldatenko, the same thing applies to the other components as well, e.g. the grafana configmap. I just didn't want to spam tons of comments.